components/dashboard/dashboard.go

```diff
@@ -48,6 +48,9 @@ type Dashboard struct {
 }
 
 func (d *Dashboard) OverrideManifests(platform string) error {
+	if d.DevFlags == nil {
+		return nil
+	}
```

Comment:
why add the logic inside of the OverrideManifests() but not from the caller?  e.g in line 102?

Reply:
hmm, so this logic cannot be added to the reconciler right? I mean it will return nil for reconciler and not proceed. I might have misunderstood your comment 😅  so please let me know how you want it

Reply:
maybe i was not clear at first:

```
if d.DevFlags != nil {
      if err = c.OverrideManifests(string(platform)); err != nil {
			return err
  }
}
```

Reply:
oh that can be definitely done. should I change it everywhere?

Reply:
updated accordingly now 👍